### DISCUSSION TITLE
how can I fix this?

### DISCUSSION THREAD

**Discussion by MohammadAboulEla** [lllyasviel/stable-diffusion-webui-forge#625] [Q&A]
WARNING:modules.face_restoration_utils:Unable to load face-restoration model
Traceback (most recent call last):
  File "D:\programing\Stable Diffusion\stable-diffusion-webui forge\webui\modules\face_restoration_utils.py", line 151, in restore_with_helper
    self.net = self.load_net()
  File "D:\programing\Stable Diffusion\stable-diffusion-webui forge\webui\modules\gfpgan_model.py", line 46, in load_net
    raise ValueError("No GFPGAN model found")
ValueError: No GFPGAN model found
08:50:30 - ReActor - STATUS - Upscaling with 4x_NMKD-Siax_200k scale = 2

**Comment by ScreNN** [comment]
> D:\programing\Stable**SPACE**Diffusion\stable-diffusion-webui**SPACE**forge

First of all, there should be no spaces in the path to the root forge folder

Change all folder names to
D:\programing\Stable-Diffusion\stable-diffusion-webui-forge

Why do you need GFPGAN? Are you trying to restore face?

**Reply by yellowcandle** [reply]
@MohammadAboulEla not sure if you are using other extensions like [ReActor](https://github.com/Gourieff/sd-webui-reactor)  
You do have to enable both the face restoration setting in the Forge setting tab **and** those in the extensions.
Please give it a try.
